lib/cisco_node_utils/cmd_ref/tacacs_server_group.yaml

```diff
@@ -1,9 +1,13 @@
 # tacacs_server_group
 ---
-_exclude: [ios_xr]
+_template:
```

Comment:
@rkorlepa Looks like you merged latest `develop` branch into your workspace instead of latest `release_1.3.0`. Please revert this merge and merge latest release instead.